packages/eslint-plugin/docs/rules/no-top-level-material-ui-4-imports.md

```diff
@@ -0,0 +1,43 @@
+# @backstage/no-top-level-material-ui-4-imports
+
+Forbid top level import from Material UI v4 packages.
+
+## Usage
+
+Add the rules as follows, it has no options:
+
+```js
+"@backstage/no-top-level-material-ui-4-imports": ["error"]
+```
+
+## Rule Details
+
+TBD - Not sure what should go here
```

Comment:
Just a short rationale of why you'd use this rule, probably, and what the effect/benefit is after applying it. Here's als a good place to mention that it has auto-fix support.

Reply:
Thanks, totally forgot about updating this, done!